.travis.yml

```diff
@@ -3,19 +3,9 @@ sudo: false
 dist: trusty
 
 cache:
-    directories:
-        - vendor
-        - $HOME/.composer/cache
```

Comment:
Is it possible to avoid these changes?

Reply:
This is just an indenting change for consistency. I can revert it if you wish.